node-otlp/tracing.js

```diff
@@ -14,17 +16,20 @@ const provider = new NodeTracerProvider();
 provider.addSpanProcessor(
   new SimpleSpanProcessor(
     new CollectorTraceExporter({
-			serviceName: 'node-otlp',
-			url: 'api.honeycomb.io:443',
-			credentials: grpc.credentials.createSsl(),
-			metadata
-		})
+      serviceName: 'node-otlp',
+      url: 'grpc://api.honeycomb.io:443/',
```

Comment:
Interesting NodeJS is different from others SDKs that use the url scheme http https to inform whether they need to set ssl credentials or insecure ones.